contrib/images/price-feeder.dockerfile

```diff
@@ -14,6 +19,7 @@ RUN WASMVM_VERSION=$(go list -m github.com/CosmWasm/wasmvm | cut -d ' ' -f 2) &&
 # Build the binary
 RUN cd price-feeder && LEDGER_ENABLED=false BUILD_TAGS=muslc LINK_STATICALLY=true make install
 
+## Prepare the final clear binary
 FROM alpine:3.17
 RUN apk add bash curl jq
```

Comment:
We need to `ca-certificates` also for price-feeder
```
RUN apk update && apk add ca-certificates
```

Reply:
why do we need it in price feeder?

Reply:
`apk update` - apk will run it with `add`